features/bootstrap/FilesystemContext.php

```diff
@@ -35,6 +35,15 @@ public function prepWorkingDirectory()
         $this->filesystem->mkdir($this->workingDirectory);
         chdir($this->workingDirectory);
 
+        $fakeHomeDirectory = sprintf('%s/fake-home/', $this->workingDirectory);
+        $this->filesystem->mkdir($fakeHomeDirectory . '.phpspec');
+
+        if (!empty($_SERVER['HOMEDRIVE']) && !empty($_SERVER['HOMEPATH'])) {
+            $_SERVER['HOMEPATH'] = substr($fakeHomeDirectory, 2);
```

Comment:
This is removing the drive letter, which I have been told is always one letter. It's a bit crude but the `pathinfo` function isn't able to do this, an alternative option might be to replace the first instance of `HOMEDRIVE` within `HOMEPATH` with an empty string, if that's preferred.